doc/api/tls.md

```diff
@@ -1154,8 +1154,7 @@ added: v0.11.13
 -->
 
 The default curve name to use for ECDH key agreement in a tls server. The
-default value is `'prime256v1'` (NIST P-256). Consult [RFC 4492] and
-[FIPS.186-4] for more details.
+default value is `'auto'`. OpenSSL will choose the curve automatically, see the [OpenSSL set curve documentation].
```

Comment:
This line should not exceed 80 characters if possible. Also, the description of the `ecdhCurve` option in `tls.createSecureContext` already covers what `auto` means, I would prefer to link to that instead of the OpenSSL documentation. Maybe something like

```markdown
default value is `'auto'`. See [`tls.createSecureContext()`] for further information.
```

I guess most users will assume this behavior, `auto` is a pretty self-explanatory name.